src/librustc_resolve/diagnostics.rs

```diff
@@ -843,6 +843,15 @@ Or:
 let unknown_variable = 12u32;
 let x = unknown_variable; // ok!
 ```
+
+Please verify that the name is declared by `use` item and ensure it's public
```

Comment:
I'd say: "If the item isn't available in the current module, please check you imported it:".